### PR TITLE
Add new idea

### PR DESCRIPTION
**Pull request by ZorianaPro**

### CODE REVIEW

File: src/components/AddNewIdea/AddNewIdea.test..js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import React from 'react';
```
Comment (yoanad) [inline src/components/AddNewIdea/AddNewIdea.test..js:1]: Your file has two `..` in the name, maybe a typo?

File: src/components/AddNewIdea/index.js [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+			.then(() => {
+				setTimeout(() => {
+					dispatch(close());
+					setShowThankYou(false);
+				}, 3000)
+			}).catch((err) => {
+				setError(err)
+			})
+		} else {
+
```
Comment (yoanad) [inline src/components/AddNewIdea/index.js:104]: Do you wanna do something here?

File: src/components/AddNewIdea/index.js [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+		(target) => {
+			const field = target.name;
+			const value = target.value;
+			switch (field) {
+				case 'firstName':
+					return setFirstName(value);
+				case 'lastName':
+					return setLastName(value);
+				case 'email':
+					return setEmail(value);
```
Comment (yoanad) [inline src/components/AddNewIdea/index.js:42]: if we are adding emails, we should probably prevent crawling :D

File: src/components/AddNewIdea/AddNewIdea.test..js (hunk 1 above)
Comment (ZorianaPro) [reply src/components/AddNewIdea/AddNewIdea.test..js:1]: yes! Thank you)))

**Pull request merged**